CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+# How to contribute
+
+Operator Marketplace is Apache 2.0 licensed and accepts contributions via GitHub pull requests. This document outlines some of the conventions on commit message formatting, instructions on how to set up a dev environment, and other resources to help get contributions into operator-marketplace.  
+
+## Setting up a dev environment
+
+- Fork the repository on GitHub
+- Clone the forked repository in your go path.
+- Get access to an Openshift cluster. The marketplace-operator is currently supported on Openshift 4.0. You can find instructions on installing an Openshift cluster [here](https://github.com/openshift/installer).
+- The Cluster-Version-Operator(CVO) manages the lifecycle of the marketplace-operator along with other internal Openshift components. To delete the marketplace-operator deployment, insruct CVO to stop managing the marketplace-operator by applying the configuration file that can be found [here](examples/cvo.override.yaml). 
+- Delete the marketplace-operator deployment `oc delete deployment openshift-marketplace -n marketplace-operator`.
+- Install [Operator-SDK](https://github.com/operator-framework/operator-sdk).
+- `operator-sdk up local --namespace=openshift-marketplace --kubeconfig=<path-to-kubeconfig-file>` to complie the marketplace-operator and start the operator in your dev environment. 
+
+### Testing changes to RBAC policy
+- If there's any modification in existing RBAC poilicies, the modifications need to be tested using a seperate deployment. Easiest way to do this is to do a local `docker build` and then push the image to a registry. Replace the [image here](https://github.com/operator-framework/operator-marketplace/blob/master/deploy/operator.yaml#L23) with the newly built image, then do a `oc apply -f deploy/` to reinstall the operator.  
+
```

Comment:
Can you specifically call out to use the `Dockerfile` in the root of the repo here? There are several dockerfiles in the project so it could be confusing.